openapi.yaml

```diff
@@ -25,105 +32,433 @@ security:
   - OAuth2:
       - full
 paths:
-  /pets:
+  /pizzas:
     get:
-      summary: List all pets
+      description: Get all pizzas
       tags:
-        - pets
-      description: Get all pets
-      operationId: getPets
+        - pizza
       parameters:
-        - $ref: '#/parameters/pageNumber'
-        - $ref: '#/parameters/pageSize'
         - in: query
-          name: species
-          description: Search by pet species
+          name: name
```

Comment:
For filter parameters, JSON:API convention is to use `filter[<name>]` where `<name>` usually corresponds to an attribute name. https://github.com/osu-mist/style-guides/blob/master/OpenAPI.md#filter-query-parameters. Most of our APIs don't follow this convention yet though.